tracer/build/_build/Build.Steps.cs

```diff
@@ -217,23 +217,13 @@ Project[] ClrProfilerIntegrationTests
                 return;
             }
 
-            if (IsWin)
-            {
-                NuGetTasks.NuGetRestore(s => s
-                    .SetTargetPath(Solution)
-                    .SetVerbosity(NuGetVerbosity.Normal)
-                    .When(!string.IsNullOrEmpty(NugetPackageDirectory), o =>
-                        o.SetPackagesDirectory(NugetPackageDirectory)));
-            }
-            else
-            {
-                DotNetRestore(s => s
-                    .SetProjectFile(Solution)
-                    .SetVerbosity(DotNetVerbosity.Normal)
-                    .SetProperty("configuration", BuildConfiguration.ToString())
-                    .When(!string.IsNullOrEmpty(NugetPackageDirectory), o =>
-                        o.SetPackageDirectory(NugetPackageDirectory)));
-            }
+            DotNetRestore(s => s
```

Comment:
Unfortunately, this is almost certainly a no-go 🙁 I've tried this several times, but tl;dr; you get issues later if you go this way does it not work with NuGet Restore? 🤔